conftest.py

```
@@ -724,24 +724,20 @@ def plan(
-            "module": pod.__module__,
-        })
+        try:
+            vm_obj = VirtualMachine(
+                client=ocp_admin_client,
+                name=vm["name"],
+                namespace=target_namespace,
+            )
+            if vm_obj.exists:
+                LOGGER.info(f"Deleting VM {vm_obj.name} after test")
+                vm_obj.clean_up(wait=True)
+        except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)